openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -689,6 +727,7 @@ A Client makes a Credential Request to the Credential Endpoint by sending the fo
 * `format`: REQUIRED. Format of the Credential to be issued. This Credential format identifier determines further parameters required to determine the type and (optionally) the content of the credential to be issued. Credential Format Profiles consisting of the Credential format specific set of parameters are defined in (#format_profiles).
 * `proof`: OPTIONAL. JSON object containing proof of possession of the key material the issued Credential shall be bound to.  The `proof` object MUST contain a following claim:
     * `proof_type`: REQUIRED. JSON string denoting the key proof type. The value of this claim determines other claims in the key proof object and its respective processing rules. Key proof types defined in this specification can be found in (#proof_types).
+* `identifier`: OPTIONAL. JSON string that identifies a credential that is being requested to be issued. It MUST be present if Credential Issuer returned `identifiers` parameter the Token Response.
```

Comment:
Not sure this parameter should be mutual exclusive with all additional parameters used to determine the type of credentials as this data was already passed in the authorization request.

Reply:
I rephrase my feedback: I suggest to change the text for 'format' so it MUST NOT be specified if it can already be determined from the identifier.

Reply:
> When using identifier on a credential request, none of these parameters should be present in the request.

+1

Reply:
```suggestion
* `identifier`: OPTIONAL. JSON string that identifies a credential that is being requested to be issued. It MUST be present if Credential Issuer returned `identifiers` parameter the Token Response. When this parameter is used, `format` parameter and any other credential format specific set of parameters such as those defined in (#format_profiles) MUST NOT be present.
```

`format` should not be present, too, right?